Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -576,6 +569,12 @@ public void StopServer()
         /// <summary>Stops and disconnects the client.</summary>
         public void StopClient()
         {
+            // return if already stopped to avoid recursion deadlock
+            if (!NetworkClient.active)
```

Comment:
why is this here?
that looks like it would be a separate PR or bug fix?

Reply:
You removed this code after this PR was made and it seems GitHub thinks this PR adds it back. Will remove.
https://github.com/vis2k/Mirror/commit/24033c101b328bb46b4bada5a7105f81d304fce1